project/MimaExcludes.scala

```diff
@@ -153,6 +153,11 @@ object MimaExcludes {
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.RealClock"),
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.Clock"),
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.TestClock")
+          ) ++ Seq(
+            // SPARK-6051
```

Comment:
This should not be required as  as ReliableKafkaReceiver is hidden. But i guess that mima binary checker is perfect.